src/bootstrap-suggest.js

```diff
@@ -288,27 +290,31 @@
             if (dataItem instanceof Object) {
                 _item.text = dataItem.text || '';
                 _item.value = dataItem.value || '';
+                _item.class = dataItem.class || '';
             } else {
                 _item.text = dataItem;
                 _item.value = dataItem;
+                _item.class = dataItem;
```

Comment:
this will add the text/value as class. remove this please

Reply:
Removed as suggested.